docs/Using-Fleet/REST-API.md

```diff
@@ -889,27 +892,6 @@ None.
      "tier": "free",
      "expiration": "0001-01-01T00:00:00Z"
    },
-  "logging": {
```

Comment:
Was that on purpose? It doesn't seem related to the PR and AFAICT we still return that logging section?

Reply:
Ah I missed that, thanks!